protocol-designer/src/components/DeckSetup/LabwareOnDeck.tsx

```diff
@@ -1,72 +1,48 @@
 import * as React from 'react'
-import { connect } from 'react-redux'
-import { LabwareRender, WellGroup } from '@opentrons/components'
+import { useSelector } from 'react-redux'
+import { LabwareRender } from '@opentrons/components'
 
 import { selectors } from '../../labware-ingred/selectors'
 import * as wellContentsSelectors from '../../top-selectors/well-contents'
 import * as highlightSelectors from '../../top-selectors/substep-highlight'
 import * as tipContentsSelectors from '../../top-selectors/tip-contents'
 import { LabwareOnDeck as LabwareOnDeckType } from '../../step-forms'
-import { ContentsByWell } from '../../labware-ingred/types'
-import { BaseState } from '../../types'
 import { wellFillFromWellContents } from '../labware/utils'
 
-interface OP {
+interface LabwareOnDeckProps {
   className?: string
   labwareOnDeck: LabwareOnDeckType
   x: number
   y: number
 }
 
-interface SP {
-  wellContents: ContentsByWell
-  liquidDisplayColors: string[]
-  missingTips?: WellGroup | null
-  highlightedWells?: WellGroup | null
-}
-
-type Props = OP & SP
-
-const LabwareOnDeckComponent = (props: Props): JSX.Element => (
-  <g
-    transform={`translate(${props.x}, ${props.y})`}
-    className={props.className}
-  >
-    <LabwareRender
-      definition={props.labwareOnDeck.def}
-      wellFill={wellFillFromWellContents(
-        props.wellContents,
-        props.liquidDisplayColors
-      )}
-      highlightedWells={props.highlightedWells}
-      missingTips={props.missingTips}
-    />
-  </g>
-)
-
-const mapStateToProps = (state: BaseState, ownProps: OP): SP => {
-  const { labwareOnDeck } = ownProps
-
-  const missingTipsByLabwareId = tipContentsSelectors.getMissingTipsByLabwareId(
-    state
+export const LabwareOnDeck = (props: LabwareOnDeckProps): JSX.Element => {
```

Comment:
For consistency

```suggestion
export function LabwareOnDeck = (props: LabwareOnDeckProps): JSX.Element {
```

I think we follow airbnb styling guidelines and I'm not sure about pd but in app we utilize `hoisting` to create a component inside component.
https://airbnb.io/javascript/react/
 
```js
// bad
class Listing extends React.Component {
  render() {
    return <div>{this.props.hello}</div>;
  }
}

// bad (relying on function name inference is discouraged)
const Listing = ({ hello }) => (
  <div>{hello}</div>
);

// good
function Listing({ hello }) {
  return <div>{hello}</div>;
}
```

Reply:
thank you for this Koji! Very helpful. I'll refactor.